frontend/src/modules/vehicles/SellVehicle.vue

```diff
@@ -183,12 +185,12 @@ export default {
       }
     },
     showMessage(message, messageType) {
-       this.message = message;
-       this.messageType = messageType;
-       setTimeout(() => {
-          this.message = null;
-          this.messageType = null;
-          }, 5000);
+      this.message = message;
+      this.messageType = messageType;
+      setTimeout(() => {
+        this.message = null;
+        this.messageType = null;
+        }, 5000);
```

Comment:
This could also be a util function, although it being short means it might be easier to copy-paste instead.

Reply:
I have addressed this refactoring in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/105/commits/1db1a9e643aa2ca3cac80922c0d9e75fb6e90ba3)